railties/lib/rails/generators/active_model.rb

```diff
@@ -72,8 +72,8 @@ def errors
       end
 
       # DELETE destroy
-      def destroy
-        "#{name}.destroy"
+      def destroy!
```

Comment:
Let's keep this name constant and just change its implementation.

Reply:
@rafaelfranca Thank you! Done and force-pushed.